src/screens/FeedScreen/Header.js

```diff
@@ -310,7 +310,7 @@ const _renderProfileNormal = ({
                 buttonStyle={{marginLeft: 'auto', paddingBottom: 0, alignSelf: 'center'}}
                 onPress={() => refSheet.current.open()}>
                 <View style={{zIndex: 1000}}>
-                  <ElipsisIcon width={4} height={14} fill={colors.blackgrey} />
+                  <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `ElipsisIcon` component has been updated from `colors.blackgrey` to `COLORS.blackgrey`. This change is consistent with the update in the import statement.
```diff
- <ElipsisIcon width={4} height={14} fill={colors.blackgrey} />
+ <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->